docs/dev/controls/image.md

```diff
@@ -0,0 +1,63 @@
+---
+id: control-image
+title: Image
+---
+
+The `Image` control adds a image asset to your layout.
+
+*This documentation is a WIP and probably not exhaustive.*
+
+---
+
+## Usage Example
+
+This example shows how to use the `Image` control.
+
+```cs
+var exampleImage = new Image(Texture2D Icon)
+{
+      Size = new Point(40, 40),
+      Location = new Point(0, 0),
+      Parent = parentPanel
+};
+```
+
+---
+
+## Available Parameters and Arguments
+
+**Size**
+Type: *Point*
+Size of the TextBox *(optional)*
```

Comment:
Looks like some C&P.  Perhaps we can word this as just control so that we are safe to C&P?  :)

Reply:
busted xD

Reply:
Sure, I'll reword everything. Good idea!

Reply:
Nice!